include/SFML/Network/IpAddress.hpp

```diff
@@ -32,6 +32,7 @@
 #include <SFML/System/Time.hpp>
 #include <iosfwd>
 #include <optional>
+#include <string_view>
 #include <string>
```

Comment:
I think we can remove `<string>`.

Reply:
Will double-check and remove if possible.

Reply:
It is necessary for `toString`.